src/ucs/config/global_opts.c

```diff
@@ -285,6 +287,18 @@ static ucs_config_field_t ucs_global_opts_read_only_table[] = {
   "Maximal size of profiling log. New records will replace old records.",
   ucs_offsetof(ucs_global_opts_t, profile_log_size), UCS_CONFIG_TYPE_MEMUNITS},
 
+ {"RCACHE_STAT_MIN", "4k",
+  "Minimum threshold for the size distribution of registered cache regions.\n"
+  "Intermediate thresholds are power-of-2 numbers between RCACHE_STAT_MIN and\n"
+  "RCACHE_STAT_MAX.",
+  ucs_offsetof(ucs_global_opts_t, rcache_stat_min), UCS_CONFIG_TYPE_MEMUNITS},
+
+ {"RCACHE_STAT_MAX", "1m",
+  "Maximum threshold for the size distribution of registered cache regions.\n"
+  "Intermediate thresholds are power-of-2 numbers between RCACHE_STAT_MIN and\n"
+  "RCACHE_STAT_MAX.",
+  ucs_offsetof(ucs_global_opts_t, rcache_stat_max), UCS_CONFIG_TYPE_MEMUNITS},
+
```

Comment:
i think it's better to require these values to be power of 2 as well. Otherwise, if user specifies some non-pow of 2 values, they will be cut of by the nearest pow of 2 values, which is confusing

Reply:
yes, it was mentioned in https://github.com/openucx/ucx/pull/7794#discussion_r780988288

Maybe it is better to mention the behavior in the description?

Reply:
yes, we need to describe it somehow. Otherwise it may be confusing if user sets non pow of 2 values